### PR TITLE
Create age mixing extractor

### PR DESCRIPTION
**Pull request by aflag**
Closes: https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/508

### CODE REVIEW

**Comment by bobturneruk** [comment]
Thanks @aflag. I like this. I think the separate package will make it easy to harmonise with work being done by others in SCRC. I will adopt the folder structure in https://github.com/ScottishCovidResponse/simple_network_sim/pull/90. Can you add a bit more description somewhere (docstring or readme) on how to run the script i.e. make the enviornment, `python .\comix_downsampler.py`, this produces `mixing-matrix.csv` which is equivalent in structure to whatever example input file, please?

I think this needs approval from @magicicada (or anyone else who has a full understanding of whether it is doing the data munging acceptably).

**Comment by aflag** [comment]
@bobturneruk I've implemented your suggestions. Please take a lok.

File: extractors/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Extractors
+
+The scripts in this folder will extract the data from the best datasource we can find in order to have a realist run of
+the simple network sim model. They are not necessarily downloading them from the official SCRCdata resource, although
+they might be partially doing that. The resulting files from the scripts in here can be plugged into the model by
+coppying them to the appropriate folder inside `sample_input_files`.
```
Comment (bobturneruk) [inline extractors/README.md:6]: coppying->copying

File: extractors/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+We expect the scripts in here will, at some point, make it into the SCRCdata repo as post-processing scripts. However,
```
Comment (bobturneruk) [inline extractors/README.md:8]: Or equivalents in another language will be written and put in SCRCdata.

**Comment by bobturneruk** [comment]
Thanks @aflag - I appreciate the extra docs - I think they will help anyone looking in to understand what we're trying to do here. Still think it needs a quick look from someone else to check the data transformation is correct before going in.

File: extractors/README.md (hunk 1 above)
Comment (mrow84) [reply extractors/README.md:6]: presumably also realist-> realistic, unless our concerns have shifted to the artistic

Comment (bobturneruk) [reply extractors/README.md:6]: any move away from the surrealist here is to be encouraged

**Comment by bobturneruk** [comment]
Please merge, @aflag.

**Pull request merged**